userspace/sysdig/chisels/scallslower.lua

```diff
@@ -85,7 +85,7 @@ function on_init()
 	-- The -pc or -pcontainer options was supplied on the cmd line
 	if print_container then
 		print(string.format("%-23.23s %-20.20s  %-20.20s %-23.23s %-20s %s",
-							"evt.datatime",
+							"evt.datetime",
```

Comment:
This is possibly a behavior change. Google doesn't indicate significant impact.

Reply:
Good catch